solidity/contracts/test/deposit/TestDeposit.sol

```diff
@@ -243,6 +243,10 @@ contract TestDeposit is Deposit {
         return self.withdrawalRequestTime;
     }
 
+    function getCollateralizationPercentage() public view returns (uint256) {
```

Comment:
Let's actually do a PR where we expose this publicly on all deposits… Alongside a `getKeepAddress`. That'll be useful for folks who want to manage their liquidations.

Reply:
We should also really expose the liquidation thresholds publicly, since they can differ from deposit to deposit.

Reply:
good idea